### ISSUE TITLE
Is DockStation compatible with Project Photon on Fusion?

### ISSUE THREAD

**Issue by MichelDiz** [DockStation/dockstation#71]
See https://github.com/vmware/photon/wiki/Running-Project-Photon-on-Fusion

Cheers

**Comment by igor-lemon** [comment]
Hm... Not sure. I'll check and write results.

**Comment by MichelDiz** [comment]
No success?

**Comment by igor-lemon** [comment]
Hi, Michel. I'm so sorry, right now I have troubles with a job and I didn't have time to check it. I'll try to do it in near time.

**Comment by MichelDiz** [comment]
Sure

**Comment by igor-lemon** [comment]
So, I have quickly gone through by docs of Photon. And if I realized it's simple Docker image under VMWare. If it's true it should works because you can work with WMWare via Docker Machine driver.

**Comment by MichelDiz** [comment]
I thought of that too. Except that the DockStation automatically generates the VM virtual machine. Does it download a certain iso image, right? and then install it in the virtual machine itself. Generating signatures and so on.

I can make docker work by terminal. But I'm using Portainer inside Photon.

The thing is, I'm having trouble using Compose that way.

If the DockStation had a more open option to connect to this image directly (ip address). Instead of an automatic process. It would be interesting. It would also even give to anyone, who knows, a way to use with other purposes like a remote Docker on the network.

**Comment by igor-lemon** [comment]
Not quite. DockStation doesn't generate VM. The app checks existence of machine by name and if VM machine doesn't exist DockStation send a command to Docker Machine to create new VM. I think that you can try to create Photon VM manually, set some name like as "photonmachine" and then set its settings into app preferences (Driver and Name on Preferences -> Docker tab).
I guess that this can work.

Also exists second way but it will be with cut functionality. Work via remote connection. You can set a remote connection with needed preferences but you can manage only containers. Right now we don't provide work with projects for remote connections.

**Comment by igor-lemon** [comment]
@MichelDiz Do you have some news? Did you try to init it via Docker Machine?

**Comment by MichelDiz** [comment]
Hey, almost 2 years xD

I came back in a new adventure and thought to use DockStation on it.

I was able to run locally the Docker binary with this config http://www.vmtocloud.com/how-to-enable-docker-remote-api-on-photon-os/

Exposing the Docker in the VM (that I've installed in a NAS machine separated from my own machine - shared in a local network) I was able to use it, without ssh. And using /usr/local/bin/docker.

But I wasn't able to use Dock Station so far. I'll try others similars just to test the concept.
